### PR TITLE
Replace pborman/uuid with google/uuid

### PR DESCRIPTION
**Pull request by Juneezee**
**What this PR does / why we need it**:

Currently we are using two UUID packages:

1. `github.com/pborman/uuid`
2. `github.com/google/uuid`

`github.com/pborman/uuid` is no longer being actively maintained (last release was 2019). So we should just use the newer `github.com/google/uuid`.

image

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [x] PR: The PR description is expressive enough and will help future contributors
- [x] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [x] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: vendor/github.com/pborman/uuid/.travis.yml [hunk 1]
```diff
@@ -1,10 +0,0 @@
-language: go
```
Comment (dankenigsberg) [inline vendor/github.com/pborman/uuid/.travis.yml:1]: Thanks for deleting code and reducing our dependencies. I don't know which implementation is better, but having two sounds bogus.

Comment (Juneezee) [reply vendor/github.com/pborman/uuid/.travis.yml:1]: Thanks for reviewing this PR. Both `github.com/pborman/uuid` and `github.com/google/uuid` implements [RFC4122](https://datatracker.ietf.org/doc/html/rfc4122), but only `github.com/google/uuid` is still being updated regularly.

**Comment by xpivarc** [comment]
/cc @brianmcarey 
can we check we fixed the prow to take this PR into account?

**Pull request merged**